ksz8863/examples/simple_switch/pytest_simple_switch.py

```diff
@@ -0,0 +1,61 @@
+# SPDX-FileCopyrightText: 2025 Espressif Systems (Shanghai) CO LTD
+# SPDX-License-Identifier: Apache-2.0
+import os
+import sys
+
+from pytest_embedded import Dut
+from test_common import HelperFunctions
+
+# Import test_common.py from parent directory
+sys.path.insert(1, '../../')
+from test_common import SwitchSSH
+from test_common import VirtualMachineSSH
+
+IP_ADDRESS_RUNNER = os.environ.get('IP_ADDRESS_RUNNER')
+IP_ADDRESS_ENDNODE = os.environ.get('IP_ADDRESS_ENDNODE')
+IP_ADDRESS_SWITCH = os.environ.get('IP_ADDRESS_SWITCH')
+SSH_VM_USERNAME = os.environ.get('SSH_VM_USERNAME')
+SSH_VM_PASSWORD = os.environ.get('SSH_VM_PASSWORD')
+SSH_SW_USERNAME = os.environ.get('SSH_SW_USERNAME')
+SSH_SW_PASSWORD = os.environ.get('SSH_SW_PASSWORD')
+SSH_PORT = 22
+
+runner = VirtualMachineSSH('Runner')
+endnode = VirtualMachineSSH('Endnode')
+switch = SwitchSSH('Switch')
+
+def test_ksz8863_simple_switch(dut: Dut) -> None:
+    # Connect to virtual machines
+    runner.connect(IP_ADDRESS_RUNNER, SSH_VM_USERNAME, SSH_VM_PASSWORD)
+    endnode.connect(IP_ADDRESS_ENDNODE, SSH_VM_USERNAME, SSH_VM_PASSWORD)
+    switch.connect(IP_ADDRESS_SWITCH, SSH_SW_USERNAME, SSH_SW_PASSWORD)
+    # Upload test script to the VMs
+    runner.put('../../vm_test_app.py', '~/vm_test_app.py')
```

Comment:
Is this really needed? Could we rather use any "standard" tools to simulate traffic like `netcat` or `socat`?